README.md

```diff
@@ -31,7 +31,7 @@ directive (class property) using HTML5' LocalStorage.
 
 3. Use the `LocalStorage` decorator
 ```typescript
-import {LocalStorage, SessionStorage} from "angular2-localstorage/WebStorage";
+import {LocalStorage} from "angular2-localstorage/WebStorage";
```

Comment:
I see. Sorry, it was not immediately evident, the examples below seemed a little bit *separate* from that.